crates/figma_import/src/variable_utils.rs

```diff
@@ -70,7 +71,9 @@ fn create_variable_value(v: &figma_schema::VariableValue) -> VariableValue {
         figma_schema::VariableValue::Boolean(b) => VariableValue::Bool(b.clone()),
         figma_schema::VariableValue::Float(f) => VariableValue::Number(f.clone()),
         figma_schema::VariableValue::String(s) => VariableValue::Text(s.clone()),
-        figma_schema::VariableValue::Color(c) => VariableValue::Color(c.into()),
+        figma_schema::VariableValue::Color(c) => VariableValue::Color(
+            (&(<&figma_schema::FigmaColor as Into<FloatColor>>::into(c))).into(),
+        ),
```

Comment:
This goes FigmaColor -> FloatColor -> Color ? Can we just keep the implementation of Into<Color> for &FigmaColor ?

Reply:
Done.